### ISSUE TITLE
Error: cannot find module '../dialog'

### ISSUE THREAD

**Issue by lparsons** [quanglam2807/appifier#244]
On Ubuntu 16.04, after installing Todoist, I was able to to run it once. However, now every time I try to run the created app, I get:

```
module.js:472
    throw err;
    ^

Error: Cannot find module '../dialog'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (/home/lparsons/.webcatalog/apps/todoist/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.get [as dialog] (/home/lparsons/.webcatalog/apps/todoist/resources/electron.asar/browser/api/exports/electron.js:11:16)
    at process.<anonymous> (/home/lparsons/.webcatalog/apps/todoist/resources/electron.asar/browser/init.js:54:31)
    at emitOne (events.js:96:13)
    at process.emit (events.js:191:7)
    at process._fatalException (bootstrap_node.js:308:26)
```

This happens even after I "uninstall" and "reinstall" the app. Tried version 9.0.0 and 9.1.2.

**Comment by quanglam2807** [comment]
This is probably the solution: https://github.com/electron/electron/issues/8862#issuecomment-294303518. I'll issue a fix ASAP.

**Comment by quanglam2807** [comment]
I've committed the fix but for a reason, Travis keeps getting "time out" when signing the app.

**Comment by quanglam2807** [comment]
Fixes with 9.2.x
